sand/controllers/transform.py

```diff
@@ -0,0 +1,245 @@
+import json
+import re
+import sys
+import traceback
+from typing import Dict, List, Callable, Any, Union, Iterable, Tuple
+
+from flask import jsonify, request
+from flask.blueprints import Blueprint
+from RestrictedPython import compile_restricted_function, safe_globals
+from werkzeug.exceptions import BadRequest
+
+from sand.models.table import Link, Table, TableRow
+from sand.models.transform import Context, Tdata, TransformRequestPayload
+from gena.deserializer import get_dataclass_deserializer
+
+transform_bp = Blueprint("transform", "transform")
+deserializer = get_dataclass_deserializer(TransformRequestPayload, {})
```

Comment:
make this variable name more specific. transform_request_payload_deserializer is not great but better than deserializer. you can pick a better name